### PR TITLE
A doc with examples of commands

### PR DESCRIPTION
**Pull request by AlexKushnir1**
Added an example with commands that are executed by the NEAR CLI to interact with the contract

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -55,4 +55,10 @@ For more details check `User contract API` impl block in the [chain-signatures/c
 
 # Environments
 1. Mainnet: `v1.signer`
-2. Testnet: `v1.sigenr-prod.testnet`
\ No newline at end of file
+2. Testnet: `v1.signer-dev.testnet`
```
Comment (volovyks) [inline API.md]: There is no mistake, it should be `v1.sigenr-prod.testnet`.

File: API.md [hunk 2]
```diff
@@ -55,4 +55,10 @@ For more details check `User contract API` impl block in the [chain-signatures/c
...
+
+# Interact using NEAR CLI 
+
+There is an `Example Commands` in the [chain-signatures/contract/example.md](./chain-signature/contract/EXAMPLE.md) file.
+
+These commands work for the latest versions of `Near-cli`(4.0.0+). For installing near-cli [[click here]](https://github.com/near/near-cli/tree/v4.0.13?tab=readme-ov-file#setup)
```
Comment (volovyks) [inline API.md]:
```suggestion
These commands were tested on [near-cli](https://github.com/near/near-cli) 4.0.0.
```

File: integration-tests/chain-signatures/src/main.rs [hunk 3]
```diff
@@ -15,6 +25,8 @@ enum Cli {
     },
     /// Spin up dependent services but not mpc nodes
     DepServices,
+    /// Example of commands to interact with the contract
```
Comment (volovyks) [inline integration-tests/chain-signatures/src/main.rs]:
```suggestion
    /// Generate example commands to interact with the contract
```

File: integration-tests/chain-signatures/src/main.rs [hunk 4]
```diff
@@ -79,6 +91,98 @@ async fn main() -> anyhow::Result<()> {
             println!("Received Ctrl-C");
             println!("Stopped dependency services");
         }
+        Cli::ContractCommands => {
+            println!("Building a doc with example commands");
+            let path = "../../chain-signatures/contract/EXAMPLE.md";
+            let mut file = File::create(path)?;
+            let mut doc: Vec<String> = vec![];
+            let contract_account_id = AccountId::from_str("v1.signer-dev.testnet")?;
+            let caller_account_id = AccountId::from_str("v1.signer-dev.testnet")?;
```
Comment (volovyks) [inline integration-tests/chain-signatures/src/main.rs]:
```suggestion
            let caller_account_id = AccountId::from_str("caller.testnet")?;
```

**Review by volovyks — DISMISSED** [review]
LGTM!
Just a few nit-picks.

**Review by ppca — DISMISSED** [review]

**Review by volovyks — APPROVED** [review]

**Pull request merged**